reporting/sending_to_dhis2.md

```diff
@@ -0,0 +1,20 @@
+### Sending Reports to DHIS2
+
+1. Set the DHIS2 UUID for the facility that is sending the data by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemr.
+     > ![](../assets/facilityUUID.PNG)
+
+2. Set the DHIS2 Server URL, username and password by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemrsync
```

Comment:
@slubwama Steps 1 and 2 can be combined to state, Configure UgandaEMR to send data to DHIS2 then have a step for adding DHIS2 UUId and the DHIS2 username and password 

How does one test that the credentials are working fine?

Reply:
@ssmusoke these I found just moved them from one big file to separate ones. The testing part is what I don't know. 
I can make changes related to formats on this one but not the reasoning around it.

Reply:
@slubwama Then add a Task in Asana so that we get that covered to ensure that each site that is setup works well